lib/puppet-strings/markdown/data_types.rb

```diff
@@ -3,41 +3,19 @@
 require_relative 'data_type'
 
 module PuppetStrings::Markdown
+  # Adapter to get information about data types
   module DataTypes
```

Comment:
I was really wondering if this class shouldn't be killed off instead. Just merge it into `DataType`. Possibly better for a future PR.

Reply:
Ah ha! I’d been thinking about different ways to clean this up, but for some reason that didn’t occur to me. It makes it a lot cleaner.